src/components/Charts.tsx

```diff
@@ -2,7 +2,7 @@
  *
  * @file Contains the component that paints Charts. Gets data for chart.
  */
-
+// eslint-disable-next-line
```

Comment:
Remove

Reply:
Currently this file contains two classes, how should I move this class? I don't think we have any place where we put classes? I can move it, but I want to know where it is ok to place a mostly empty class in the hierarchy